connectors/sources/tests/test_generic_database.py

```diff
@@ -25,6 +26,18 @@
 ORACLE_CONNECTION_STRING = "oracle+oracledb://admin:changme@127.0.0.1:1521/testdb"
 
 
+class mock_engine:
+    """This Class create mock engine for mssql dialect"""
```

Comment:
Why does this class live in `test_generic_database.py`, if it's specific for MSSQL?